pkg/specgen/namespaces.go

```diff
@@ -443,6 +443,19 @@ func ParseNetworkFlag(networks []string, pastaNetworkNameExists bool) (Namespace
 		}
 	}
 
+	// Set unnamed interaces' names to their parent interface names as per
```

Comment:
@Luap99 , @rhatdan ,
Does this place sound correct for the change to handle podman container create/run and podman network connect?
Also, any clue for how to extract parent interface name here?

Reply:
This is not the correct place as this will run on the remote client and we should only use this containers.conf field on the server side.
You have to do this at the place where we already assign the ethX names. https://github.com/containers/podman/blob/1fc34d8e2f94a67270ec6cb09fbcdac00f61538d/libpod/runtime_ctr.go#L272

We already call normalizeNetworkName() there which inspects the network and the NetworkInterface field should contain the bridge/macvlan parent name.

Reply:
Thanks @Luap99 . Will check that.

Reply:
Made the changes accordingly.